frameworks/CSharp/carter/README.md

```diff
@@ -0,0 +1,26 @@
+# Carter Tests on Linux
+This includes tests for plaintext and json serialization.
+
+## Infrastructure Software Versions
+
+**Language**
+
+* C# 7.0
```

Comment:
Does this need to be 7.3?

Reply:
Well async main is only 7.0; I just set the project to 7.3 out of habit :)